src/index/getFilePaths.ts

```diff
@@ -19,7 +19,7 @@ const globSearch = (pattern: string) => {
 };
 
 /** Recursively get all file paths. */
-export const getFilePaths = (paths: string[]) => {
+export const getFilePaths = (paths: string[], _options: {}) => {
```

Comment:
Instead of adding `_options` I think we should remove passing in options to the `getFilePaths` function.

Reply:
done, I thought perhaps it was going to be used later

Reply:
👌

Could you also remove `noEmit` from the check script? It is already set in this tsconfig.